view/adminhtml/templates/order/invoice/view/form.phtml

```diff
@@ -62,24 +62,26 @@ The order can be canceled from the Balance dashboard.')); ?></div>
                     <div class="shipping-description-content">
                         <?= $block->escapeHtml(__('Total Shipping Charges')) ?>:
 
-                <?php if ($block->helper(\Magento\Tax\Helper\Data::class)->displayShippingPriceIncludingTax()): ?>
-                    <?php $_excl = $block->displayShippingPriceInclTax($_order); ?>
-                <?php else: ?>
-                    <?php $_excl = $block->displayPriceAttribute('shipping_amount', false, ' '); ?>
-                <?php endif; ?>
-                <?php $_incl = $block->displayShippingPriceInclTax($_order); ?>
+                        <?php if ($this->helper(\Magento\Tax\Helper\Data::class)
```

Comment:
this vs block, a scoping error?

Reply:
It's just an error, $block never had the 'helper' method so it crashed.

This PR was supposed to include only CS-fixer fixes such as formatting, linting, etc.